java-10/appdynamics/README.md

```diff
@@ -8,11 +8,45 @@ Se README for Java 10 for general usage.
 
 ## Configuration of AppDynamics
 
+### Application name
 AppDynamics will automatically report itself by using the value of the
-environment variable `APP_NAME` as the Application name.
+environment variable `APPD_NAME` as the Application name.
 
-To use a different Application name in AppDynamics, the environment variable
-`APPD_NAME` can be set in your Dockerfile.
+```
+ENV APPD_NAME=my_app
+```
+
+The most generic way use this is to forward what NAIS injects as `APP_NAME`
+by adding the following to your Dockerfile:
+
+```
+ENV APPD_NAME=APP_NAME
```

Comment:
Will this work? Isn't ENV is set build time?

Reply:
Indeed, but I'm doing a neat/dirty trick with bash, and do a indirect variable expansion like this: https://github.com/navikt/baseimages/pull/37/files#diff-cc2ae8d7c247403f3fc805e503a9de78R5

If the value maps to another environment variable that gets used, otherwise it will be treated like a string.

It's a bit icky. But prevent the need for a separate init-script to be added.

Reply:
Could add another flag like `APPD_ENABLED` and simply use `APP_NAME` as the default unless otherwise overwritten.

Reply:
Ah. 👍